### ISSUE TITLE
[Bug] cannot import carto module using script tag

### ISSUE THREAD

**Issue by felixpalmer** [visgl/deck.gl#7745]
### Description

`8.9.1` introduced a packaging bug (most likely due to https://github.com/visgl/deck.gl/pull/7741) where `registerLoaders` cannot be resolved from loaders.gl:

```
dist.min.js:24 Uncaught TypeError: (0 , As.registerLoaders) is not a function
    at dist.min.js:24:4977
    at dist.min.js:29:1651
    at webpackUniversalModuleDefinition (dist.min.js:6:23)
    at dist.min.js:6:35
```

### Flavors

- [ ] React
- [ ] Python/Jupyter notebook
- [ ] MapboxLayer
- [ ] GoogleMapsOverlay
- [ ] CartoLayer
- [ ] DeckLayer/DeckRenderer for ArcGIS

### Steps to Reproduce

https://jsfiddle.net/felixp/bpz6mwud/16/

### Environment

- Framework version: 8.9.1
- Browser: all
- OS: all

**Comment by Pessimistress** [comment]
Looks like the globals setting is missing a reference for`@loaders.gl/core`: https://github.com/visgl/deck.gl/blob/master/.ocularrc.js#L32-L36

**Comment by Pessimistress** [comment]
Fixed in 8.9.2
